src/test/app/forms/models/address.ts

```diff
@@ -148,6 +148,37 @@ describe('Address/CorrespondenceAddress', () => {
 
         expect(errors.length).to.equal(0)
       })
+
+      context('addressVisible is false and addressSelectorVisible is false', () => {
```

Comment:
I would love more plain English version describing how the scenario triggered e.g. `when postcode lookup has not been triggered` or something.

Would be also good to have assertion on the expected message, nut just the number of errors.